.gitignore

```diff
@@ -9,4 +9,5 @@ CWAGENT_VERSION
 .terraform.*
 terraform.*
 .terraform/*
+**/.terraform/*
```

Comment:
I think if we have this, we don't need the line above, `.terraform/*`, since this encompasses that.

Reply:
Removed